layers/+source-control/git/packages.el

```diff
@@ -112,13 +112,21 @@
 
       (evil-leader/set-key
         "gb" 'spacemacs/git-blame-micro-state
-        "gc" 'magit-commit
+        "gc" 'magit-commit-popup
         "gC" 'magit-checkout
-        "gdh" 'spacemacs/magit-diff-head
+        "gd" 'magit-diff-popup
+        "gD" 'spacemacs/magit-diff-head
         "gi" 'magit-init
-        "gl" 'magit-log-all
+        "gl" 'magit-log-popup
         "gL" 'magit-log-buffer-file
-        "gs" 'magit-status)
+        "gf" 'magit-fetch-popup
+        "gF" 'magit-pull-popup
+        "gP" 'magit-push-popup
+        "gs" 'magit-status
+        "gw" 'magit-stage-file
+        "gu" 'magit-unstage-file
+        "gE" 'magit-ediff-compare
+        "ge" 'magit-ediff-show-working-tree)
```

Comment:
Changes here seem unrelated to vcs micro state. I assume you don't want to change the spacemacs' default bindings in this PR. Probably a mistake?